### PR TITLE
Adds empty sample app module + code formatting

### PR DESCRIPTION
**Pull request by felipecsl**

### CODE REVIEW

File: apollo-compiler/src/main/kotlin/com/apollostack/compiler/GraphqlCompiler.kt [hunk 1]
```diff
@@ -28,12 +28,9 @@ open class GraphqlCompiler {
 
   private fun String.formatPackageName(): String {
     val parts = split(File.separatorChar)
-
-    for (i in 2..parts.size) {
-      if (parts[i - 2] == "src" && parts[i] == "graphql") {
-        return parts.subList(i + 1, parts.size).dropLast(1).joinToString(".")
-      }
-    }
+    (2..parts.size)
```
Comment (felipecsl) [inline apollo-compiler/src/main/kotlin/com/apollostack/compiler/GraphqlCompiler.kt]: @sav007 turns out this is the code you had originally haha, Android Studio suggested the auto refactoring

Comment (sav007) [inline apollo-compiler/src/main/kotlin/com/apollostack/compiler/GraphqlCompiler.kt]: Yeah :)

**Comment by sav007** [comment]
Hey could you please extract your code style formatter and put it under git, that we all will be at the same page. And can we increase margin from 100 -> 120 please

**Comment by felipecsl** [comment]
@sav007 i did it already, it's the file `.idea/codeStyleSettings.xml` in this PR
I'll increase the margin to 120

File: apollo-compiler/src/main/kotlin/com/apollostack/compiler/GraphqlCompiler.kt (hunk 1 above)
Comment (marwanad) [inline apollo-compiler/src/main/kotlin/com/apollostack/compiler/GraphqlCompiler.kt]: What do you think about `parts.size - 1 downTo 2` instead? 
It doesn't really matter but that should handle all the cases now 😀 (including the `src/foo/grahpql/src/bar/graphql/Query.json`)

Comment (felipecsl) [inline apollo-compiler/src/main/kotlin/com/apollostack/compiler/GraphqlCompiler.kt]: sounds good to me

**Comment by felipecsl** [comment]
@sav007 updated, PTAL

**Pull request merged**